### ISSUE TITLE
Deleting Comments

### ISSUE THREAD

**Issue by beeterry** [mlarhrouch/azure-pipeline-gpt-pr-review#49]
This works great. I have this triggered in a build policy, so as changes get pushed to a PR, this gets run again. I see that the code exists to delete comments, but in my case, the existing comments from the prior changes aren't deleted. I would expect the comments prior to the changes would be deleted. Is this a bug in the codebase or do I need to enable some other permission?

**Comment by krclarke101** [comment]
Hi there,

Did the review actually create any comments? 

I just get this 

Start deleting existing comments added by the previous Job ...
Existing comments deleted.
Start reviewing

**Comment by beeterry** [comment]
Yes it creates comments. I struggled with getting the permissions right there too. You need to go to your repository security tab where the user permissions are. Then search for "Project collection build service" and a user should appear. You then have to set the Contribute and Contribute to pull requests to "allow".

**Comment by krclarke101** [comment]
Hi, thanks for the info. I added allow to the Contribute but I can't find "Contribute to pull requests" was it in the same place?

**Comment by beeterry** [comment]
Yes they are right next to each other. Mine looked as screenshotted in the docs of the repo.
